### PR TITLE
[N/A] Update service tooling

### PR DESCRIPTION
**Pull request by bryangaleopenfin**

### CODE REVIEW

File: hooks.ts [hunk 1]
```diff
@@ -1,10 +1,12 @@
-const fs = require('fs');
-const path = require('path');
+import fs = require('fs');
```
Comment (tomrobertsOF) [inline hooks.ts]: I've never seen this syntax before. Does this actually work?

File: hooks.ts [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
+import path = require('path');
 
-module.exports = (app) => {
+import {Hook, registerHook} from 'openfin-service-tooling/utils/allowHook';
+
+registerHook(Hook.TEST_MIDDLEWARE, (app) => {
     // Sneakily return the test directory instead of the default one
     app.get('/provider/sample-app-directory.json', (req, res) => {
-        const testDirectory = JSON.parse(fs.readFileSync(path.join('res', 'test', 'sample-app-directory.json')));
+        const testDirectory = JSON.parse(fs.readFileSync(path.join('res', 'test', 'sample-app-directory.json')).toString());
```
Comment (tomrobertsOF) [inline hooks.ts:9]: Why is this `toString` here? Wouldn't that just return `[Object object]` or something like that?

Comment (bryangaleopenfin) [reply hooks.ts:9]: Without it I get "Argument of type 'Buffer' is not assignable to parameter of type 'string'.". Will double check it's doing what I assume

File: hooks.ts (hunk 1 above)
Comment (bryangaleopenfin) [reply hooks.ts]: It seems?! Copied from the `os-launchpad` `hooks.ts` file

Comment (bryangaleOF) [reply hooks.ts]: Yep, does what I expected. Which makes it werid that it worked in the first place. I think JSON.parse must specifically check for buffer objects and handle appropriately, but that's not reflected in the types

File: hooks.ts (hunk 2 above)
Comment (bryangaleopenfin) [reply hooks.ts:9]: Yep, does what I expected. Which makes it werid that it worked in the first place. I think JSON.parse must specifically check for buffer objects and handle appropriately, but that's not reflected in the types

Comment (tomrobertsOF) [reply hooks.ts:9]: Hmm, fair enough.

File: hooks.ts (hunk 1 above)
Comment (tomrobertsOF) [reply hooks.ts]: Hmmm. I really don't like it, but if it works then ok I guess.

**Review by tomrobertsOF — APPROVED** [review]

**Review by ottah — APPROVED** [review]

**Pull request merged**